### ISSUE TITLE
about constructing training set.

### ISSUE THREAD

**Issue by Sang-Yeop-Yeo** [hkchengrex/MMAudio#50]
First of all, I sincerely congratulate you on your excellent research is accepted to CVPR 2025.

I would like to learn more about the training data used for your model.

Specifically, the VGGSound dataset provides a clear separation between the train set and the test set.

However, during my inference on the VGGSound test set, I found that the sample 3feU6jqOjnU (class: zebra braying) was almost identical to the real data.

I suspect that such a high degree of similarity is due to memorization.
In other words, I believe that this test set sample was included in the training process.

Did you merge the training and test sets and then re-split them to form a new test set?

If so, how many samples are included in the test set?

Thank you for your time in addressing my questions.
I wish you the best of luck.
Thank you.

**Comment by hkchengrex** [comment]
Hello, thanks for the question!

**We use the original train-test split of VGGSound.** This issue happens because the same video is sometimes uploaded to YouTube multiple times with different video IDs, and VGGSound did not de-duplicate them. If a video sounds exactly the same as the ground-truth, there is a chance that the same video is in the training set under a different video ID. This is not ideal, but I believe the portion of these videos is small and this is what all prior works have done (e.g., training on the train set).

For example, this video is in the test set: https://www.youtube.com/watch?v=FhaYvI1yrUM
And these two videos are in the training set: https://www.youtube.com/watch?v=vHmRikW9axQ, https://www.youtube.com/watch?v=1MQkMdlBezY

I hope this answers your question.

**Comment by Sang-Yeop-Yeo** [comment]
This is a situation I never expected!

Thank you for your kind response.
It was very helpful.
